src/caselawclient/models/documents/__init__.py

```diff
@@ -171,13 +172,11 @@ def _initialise_identifiers(self) -> None:
         self.identifiers = unpack_all_identifiers_from_etree(identifiers_element_as_etree)
 
     @property
-    def best_human_identifier(self) -> Optional[str]:
-        """
-        Some identifier that is understood by legal professionals to refer to this legal event
-        that is not the name of the document.
-        Typically, this will be the neutral citation number, should it exist.
-        Should typically be overridden in subclasses.
-        """
+    def best_human_identifier(self) -> Optional[Identifier]:
```

Comment:
Do all identifiers have a sensible `__str__` such that this is a good drop-in replacement despite the type change?

Reply:
No, but I want this to explode where we're not using it properly. This is definitely a breaking change.